var/spack/repos/builtin/packages/slate/package.py

```diff
@@ -89,7 +89,8 @@ class Slate(CMakePackage, CudaPackage, ROCmPackage):
     depends_on("lapackpp@2021.04.00:", when="@2021.05.01:")
     depends_on("lapackpp@2020.10.02", when="@2020.10.00")
     depends_on("lapackpp@master", when="@master")
-    depends_on("scalapack", type="test")
+    depends_on("scalapack", when="@:2022.07.00", type="test")
+    depends_on("python", type="test")
```

Comment:
Does this dependency applies to all listed versions of the package?

Reply:
The python version does.  The tester script has always used python.  I just noticed that it fails on a system when python is not installed.